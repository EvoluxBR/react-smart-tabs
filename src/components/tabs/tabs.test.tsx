import * as React from 'react';
import { shallow } from "enzyme";
import Tab from "./tab";
import TabBar from "./tab-bar";

const component = <h1>Test</h1>;
describe('tab subcomponent', () => {
  it("should render a Tab", () => {
    const result = shallow(
      <Tab id={'1'} text={'firstTab'}>
        <h1>Test</h1>
      </Tab>
    );
    expect(result).toMatchSnapshot();
  });
});

describe('tab subcomponent', () => {
  it("should render a tab bar with one tab inside", () => {
    const result = shallow(
      <TabBar>
        <Tab id={'1'} text={'firstTab'}>
          <h1>Test</h1>
        </Tab>;
      </TabBar>
    );
    expect(result).toMatchSnapshot();
  });
  // it("should only display the active tab") {
  //   const result = shallow(
  //     <TabBar>
  //       <Tab id={'1'} text={'firstTab'} className="active">
  //         <h1>Test</h1>
  //       </Tab>;
  //       <Tab id={'2'} text={'firstTab'} >
  //         <h1>Test</h1>
  //       </Tab>);
  //   </TabBar>
  //   );
  // }
});
