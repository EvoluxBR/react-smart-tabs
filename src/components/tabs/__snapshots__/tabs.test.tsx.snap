// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tab subcomponent should render a Tab 1`] = `ShallowWrapper {}`;

exports[`tab subcomponent should render a tab bar with one tab inside 1`] = `ShallowWrapper {}`;
